docs/oscar_references.bib

```diff
@@ -678,6 +691,15 @@ @InCollection{Ste91
   url           = {https://doi.org/10.1007/BFb0086390}
 }
 
+@Book{Stu93,
+  author        = {Sturmfels, Bernd},
+  title         = {Algorithms in invariant theory},
+  series        = {Texts and Monographs in Symbolic Computation},
+  publisher     = {Springer-Verlag, Vienna},
+  year          = {1993},
+  pages         = {vi+197}
```

Comment:
```suggestion
  pages         = {vi+197},
  doi           = {10.1007/978-3-7091-4368-1},
  url           = {https://doi.org/10.1007/978-3-7091-4368-1}
```
Ideally also specify the DOI and URL then the reference becomes clickable in the online manual (of course we are also missing this this in many other bib items, so... *shrug*